tensorflow/contrib/lite/experimental/c/c_api.h

```diff
@@ -167,6 +167,10 @@ TFL_CAPI_EXPORT extern int32_t TFL_InterpreterGetOutputTensorCount(
 TFL_CAPI_EXPORT extern const TFL_Tensor* TFL_InterpreterGetOutputTensor(
     const TFL_Interpreter* interpreter, int32_t output_index);
 
+// Prints a dump of what tensors and what nodes are in the interpreter.
```

Comment:
I don't know that we want to put this in the public stable API quite yet, what about moving to c_api_experimental.h?